domino/domino.py

```diff
@@ -238,8 +238,14 @@ def runs_stdout(self, runId):
         # pprint.pformat outputs a string that is ready to be printed
         return pprint.pformat(self._get(url)['stdout'])
 
-    def job_start(self, command: str, commit_id: str = None, hardware_tier_name: str = None,
-                  environment_id: str = None, on_demand_spark_cluster_properties: dict = None) -> dict:
+    def job_start(
+            self,
+            command: str,
+            commit_id: str = None,
+            hardware_tier_name: str = None,
+            environment_id: str = None,
+            on_demand_spark_cluster_properties: dict = None,
+            compute_cluster_properties: dict = None) -> dict:
```

Comment:
Not super-important but more "thorough" (see https://docs.python.org/3/library/typing.html#typing.Optional)
 ```suggestion
             commit_id: Optional[str] = None,
             hardware_tier_name: Optional[str] = None,
             environment_id: Optional[str] = None,
             on_demand_spark_cluster_properties: Optional[dict] = None,
             compute_cluster_properties: Optional[dict] = None) -> dict:
```

Reply:
yes, this is a good idea

Reply:
done